src/workerd/jsg/rtti.capnp

```diff
@@ -212,6 +212,16 @@ struct Structure {
   asyncIterator @7 :Method;
   # Method returning async iterator if the structure is async iterable
 
+  disposable @13 :Bool;
+  # true if the structure is disposable
+  dispose @14 :Method;
+  # dispose method
```

Comment:
Just curious, why two separate fields for each? Could we not assume `disposable` is true if `dispose` has a value?

Reply:
I was just copying the pattern from iterable and asyncIterable. No idea why it's done this way but assume I should stay consistent.